integration-tests/src/test/resources/solr6/solrconfig_insight.xml

```diff
@@ -0,0 +1,6 @@
+<xi:include href="onPurposeNonExistingFile.xml" xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
You don't need this special construct to have your changes imported.

The only requirement is that you have a single root element, but I think it can be `<config>`

Reply:
This is what the real solrconfig_insight.xml from alfresco contains. 
I kept it as it was, because it leaves room for another file, if needed.

Reply:
@vierbergenlars Otherwise is it ok? Can I merge and make a new PR to release 0.5.0?

Reply:
Yes, this looks okay otherwise